db/seeds.rb

```diff
@@ -123,3 +123,38 @@
             image: "galette.jpg")
 
 CategoryItem.create(item_id: 15, category_id: 5)
+
+Item.create(title: "Bacon Skewers",
+description: "Maple Chocolate Bacon Skewers",
+price: 900,
+image: "skewers.jpg")
+
+CategoryItem.create(item_id: 16, category_id: 5)
+
+Item.create(title: "Brownies",
+description: "Chocolate brownies smothered in vanilla icing and topped with bacon!",
```

Comment:
Line is too long. [84/80]<br>Align the elements of a hash literal if they span more than one line.